packages/ciscospark/src/plugins/messages.js

```diff
@@ -5,6 +5,7 @@
  */
 
 import {SparkPlugin, Page} from '@ciscospark/spark-core';
+import {isArray} from 'lodash';
```

Comment:
just curious, in general, is it better to use lodash's isArray, or native Array.isArray?